package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "ng serve",
     "build": "ng build --target=production --aot",
-    "coverage": "ng test --no-watch --code-coverage --single-run=true",
+    "coverage": "ng test -e prod --no-watch --code-coverage --single-run=true -sm=false",
```

Comment:
Is this necessary? If possible, I'd prefer to run tests in dev. Doesn't testing with a prod environment make the debug messages harder to read?

Also, minor nit: I prefer to use verbose flags where possible, so like `--environment prod` and `--sourcemaps=false`